Analysis/R/test_case/test_case_3.R

```diff
@@ -355,7 +355,7 @@ all_dfs$observations_df <- test_observations %>%
   )
```

Comment:
Why are you making changes to the .R files?

Reply:
my bad, this shouldn't be here